### ISSUE TITLE
working example

### ISSUE THREAD

**Issue by fariza** [cihologramas/freecad-pyoptools#4]
It would be nice to include a little working example to open it and see the possibilities

**Comment by ramezquitao** [comment]
The workbench is not stable yet. Meaning we are changing the internals a
lot, so a working example may not work for long. I think in a week or so we
may have something quasi stable so we can put some example and small
instructions


Ricardo Amézquita Orozco
Doctor en Ciencias - Física
Director de Investigación y Desarrollo

CI Hologramas - Combustión Ingenieros S.A.S.
Cll 25B # 85B - 76, Bogotá / Colombia
Tel: 57-1-2635232
Fax: 57-1-4296516
http://www.cihologramas.com

**Comment by luzpaz** [comment]
Any progress on an example ?

**Comment by ramezquitao** [comment]
No, sorry. But some small instructions on how to use it were added to the README. At least it is working with python3 now.
